docs/content/preview/explore/linear-scalability/_index.md

```diff
@@ -25,48 +25,25 @@ showRightNav: true
-
-- Easy. Scaling should be operationally simple to do.
-- Elastic. You should be able to scale up and down as needed.
-- Transparent. Your applications shouldn't even know it's happening.
-
-## How scaling works
+- **Growing user base**. Your application becomes popular, users love your app, and the user base is expanding.
+- **Seasonal traffic**. Occasionally, you have to handle a lot more transactions per second than usual. Black Friday and Cyber Monday retail traffic, or streaming for special events like the Superbowl or World Cup, for example.
+- **Growing datasets**. For example, you have an IoT app or an audit database that keeps growing rapidly daily. These systems have to handle a high volume of writes regularly.
+- **Changing business priorities**. Scaling needs are often unpredictable. To take one example, retail priorities shifted radically when Covid entered the picture. With a database that can scale, you can pivot quickly when the business environment shifts.
+- **New geographies**. Your user base expands to new regions, and you need to add to your presence globally by adding more data centers in different continents.
 
-In YugabyteDB, [data is split (sharded)](./sharding-rebalancing) into tablets, and these multiple tablets are placed on various nodes. When more [nodes are added](./node-addition), some tablets are automatically [rebalanced](./sharding-rebalancing#rebalancing) to the new nodes. Data is moved automatically, without any interruption in service. Tablets can be split dynamically as needed to use the newly added resource, which leads to each node managing fewer tablets. The entire cluster can therefore handle more transactions and queries in parallel, thus increasing its capacity to handle larger workloads.
+Being able to scale seamlessly is as important as being able to scale. Scaling needs to be operationally simple and completely transparent to the applications. With YugabyteDB, you can start small and add nodes as needed. You can scale your data, reads, and writes without disrupting ongoing applications. As your needs grow, YugabyteDB automatically shards data and scales out. You can also scale up your cluster for short-term needs and then scale down once the need is over.
 
-To learn more, see [Sharding and rebalancing](sharding-rebalancing/) and [Adding nodes](node-addition/).
+## Ways to scale
 
-## How to scale
-
-Depending on your specific requirements, you scale your database to support one or more of the following:
-
-- More transactions per second
-- More concurrent client connections
-- Larger datasets
-- Additional regions for an expanding global presence
-
-You can achieve this in two ways: vertical and horizontal scaling.
+There are 2 common ways of scaling, namely **vertical** and **horizontal**. YugabyteDB supports both vertical and horizontal scaling. Because YugabyteDB is distributed, scaling is operationally straightforward and performed without any service disruption.
```

Comment:
```suggestion
There are 2 common ways to scale, namely **vertical** and **horizontal**. YugabyteDB supports both. Because YugabyteDB is distributed, scaling is operationally straightforward and performed without any service disruption.
```